homeassistant/components/wemo/__init__.py

```diff
@@ -142,8 +148,8 @@ class WemoDispatcher:
     def __init__(self, config_entry: ConfigEntry) -> None:
         """Initialize the WemoDispatcher."""
         self._config_entry = config_entry
-        self._added_serial_numbers = set()
-        self._loaded_components = set()
+        self._added_serial_numbers: MutableSet[str] = set()
+        self._loaded_components: MutableSet[str] = set()
```

Comment:
```suggestion
        self._added_serial_numbers: set[str] = set()
        self._loaded_components: set[str] = set()
```

Can you type this as a `set` instead?